test for alpine-2.22's c-client and cope with API change.
parts borrowed from slackware patch for PHP with alpine-2.22.

Index: apps/app_voicemail.exports.in
--- apps/app_voicemail.exports.in.orig
+++ apps/app_voicemail.exports.in
@@ -10,6 +10,7 @@
 		LINKER_SYMBOL_PREFIXmm_list;
 		LINKER_SYMBOL_PREFIXmm_log;
 		LINKER_SYMBOL_PREFIXmm_login;
+		LINKER_SYMBOL_PREFIXmm_login_method;
 		LINKER_SYMBOL_PREFIXmm_lsub;
 		LINKER_SYMBOL_PREFIXmm_nocritical;
 		LINKER_SYMBOL_PREFIXmm_notify;
